MyOtherClass.cls

```diff
@@ -1,11 +1,16 @@
-public without sharing class MyClass {
+public inherited sharing class MyOtherClass {
 
-     public void violationExpected(){
-             String malicious = '32423';
-             Account account = [SELECT id,AccountId FROM Contact WHERE Id = :malicious LIMIT 1];
+     @AuraEnabled
+     public void violationExpected(String malicious){
+          //String malicious = '32423';
+          Account account = [SELECT id,AccountId FROM Contact WHERE Id = :malicious LIMIT 1];
+          // A comment
+          // Another comment
+          String p = 'a string';
+          List<Account> items = [SELECT * FROM account]; // Inline comment
```

Comment:
The result of this query is not bound. Please add either a `LIMIT` or a `WHERE` statement to the query to limit the number of results.